### ISSUE TITLE
Annotation over annotation

### ISSUE THREAD

**Issue by vsoaresbruno** [albertjuhe/annotator_view#5]
Hi, i wanna know if is possible use annotation over annotation, I looking for solution but unsuccessfully, can you help me?

**Comment by albertjuhe** [comment]
Yes, You can annotate something and annotate again in the text annotated, when your pointer is over this zone you can see two annotations one below the other.
